### ISSUE TITLE
Method of overwriting a saved preset?

### ISSUE THREAD

**Issue by milkmiruku** [drobilla/jalv#8]
I'm not sure if it's currently possible to overwrite a saved preset. If I select the directory in the list of a saved preset (or enter the directory name in the Name field) then click Save, it enters the directory.

**Comment by drobilla** [comment]
Yeah, the Gtk file dialog (and most file dialogs except on Mac) are really bad at directory selection in general.

I'm not sure how to go about this.  Sometimes I wonder if it shouldn't be file selector based at all, but simply ask for a name and save presets always to the local LV2 path which is where they need to be to really work anyway...

**Comment by drobilla** [comment]
> Sometimes I wonder if it shouldn't be file selector based at all

I've think I will just throw the file selector away, and replace it  with a simple category/preset UI, which happens to mesh well with a more modern Gtk UI with a header bar.

I doubt anyone will miss the ability to save state to a specific location, that seems like a "developerey" enough task that it doesn't need to be exposed in the GUI.  Maybe as part of a more advanced command-line interface or something.  In the meantime, worst case scenario, the bundle can just be moved easily enough.

**Comment by drobilla** [comment]
Added in 99b7959
